data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1011,7 +1011,7 @@ spec:
                     description: ClusterOSImage is either a URL with `http(s)` or `file` scheme to override the default OS image for cluster nodes, or an existing Glance image name.
                     type: string
                   computeFlavor:
-                    description: FlavorName is the name of the compute flavor to use for instances in this cluster.
+                    description: 'FlavorName is the name of the compute flavor to use for instances in this cluster. Deprecated: this value is not used by the installer, use type in the defaultMachinePlatform section to define the flavor.'
```

Comment:
I think you mean "flavorName". in installconfig.go, it reads: "config.Platform.OpenStack.DefaultMachinePlatform.FlavorName = config.Platform.OpenStack.FlavorName"

Reply:
done